README.md

```diff
@@ -1,92 +1,90 @@
-considerable amount of time. Subsequent builds are faster.
-
-Presto has a comprehensive set of unit tests that can also take a considerable
-amount of time to run. You can disable the tests when building:
+while, depending on your connection speed. Subsequent builds are faster.
 
-    ./mvnw clean install -DskipTests
+Trino has a comprehensive set of tests that take a considerable amount of time
+to run, and are thus disabled by the above command. These tests are run by the
+CI system when you submit a pull request. We recommend only running tests
+locally for the areas of code that you change.
 
-## Running Presto in your IDE
+## Running Trino in your IDE
 
 ### Overview
 
-After building Presto for the first time, you can load the project into your IDE
-and run the server. We recommend using [IntelliJ
-IDEA](http://www.jetbrains.com/idea/). Because Presto is a standard Maven
-project, you can import it into your IDE using the root `pom.xml` file. In
-IntelliJ, choose Open Project from the Quick Start box or choose Open from the
-File menu and select the root `pom.xml` file.
+After building Trino for the first time, you can load the project into your IDE
+and run the server.  We recommend using
+[IntelliJ IDEA](http://www.jetbrains.com/idea/). Because Trino is a standard
+Maven project, you easily can import it into your IDE.  In IntelliJ, choose
+*Open Project* from the *Quick Start* box or choose *Open*
+from the *File* menu and select the root `pom.xml` file.
 
 After opening the project in IntelliJ, double check that the Java SDK is
 properly configured for the project:
 
 * Open the File menu and select Project Structure
 * In the SDKs section, ensure that JDK 11 is selected (create one if none exist)
-* In the Project section, ensure the Project language level is set to 8 (Presto
-  does not yet use Java 11 language features)
+* In the Project section, ensure the Project language level is set to 11
+
+### Running a testing server
+
+The simplest way to run Trino for development is to run the `TpchQueryRunner`
+class. It will start a development version of the server that is configured with
+the TPCH connector. You can then use the CLI to execute queries against this
+server. Many other connectors have their own `*QueryRunner` class that you can
+use when working on a specific connector.
 
-Presto comes with sample configuration that should work out-of-the-box for
+### Running the full server
+
+Trino comes with sample configuration that should work out-of-the-box for
 development. Use the following options to create a run configuration:
 
 * Main Class: `io.trino.server.TrinoServer`
-* VM Options: `-ea -XX:+UseG1GC -XX:G1HeapRegionSize=32M -XX:+UseGCOverheadLimit -XX:+ExplicitGCInvokesConcurrent -Xmx2G -Dconfig=etc/config.properties -Dlog.levels-file=etc/log.properties -Djdk.attach.allowAttachSelf=true`
+* VM Options: `-ea -Dconfig=etc/config.properties -Dlog.levels-file=etc/log.properties -Djdk.attach.allowAttachSelf=true`
```

Comment:
I see you removed redundant properties that are useful in general, like `-Xmx` (useful when you start a few JVMs on your laptop). In doing so, any reason to leave `-ea`, `-Djdk.attach.allowAttachSelf=true` or `-Dlog.levels-file` here? Are they not redundant in strict sense of the word?